docs/api/openapi/agent-management.yaml

```
@@ -0,0 +1,1307 @@
+                  timeout: 300
+              code_review:
+                summary: Code review task
+                value:
+                  task: "Review API implementation"
+                  context:
+                    language: "typescript"
+                    framework: "nestjs"
+                  files:
+                    - path: "/src/users/users.controller.ts"
+                      content: "// Controller implementation"
+                    - path: "/src/users/users.service.ts" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)